From 281b314f38564537279c94e605bee74ff41b0245 Mon Sep 17 00:00:00 2001
From: Ingo Schwarze <schwarze@usta.de>
Date: Sun, 30 Aug 2015 15:13:07 +0200
Subject: [PATCH 8/8] remove bogus \/ escapes

some X manuals use then escape sequence \/ when they want to render
a slash.  That's bad because \/ is not a slash but an italic
correction, never producing any output, having no effect at all in
terminal output, and only changing spacing in a minor way in typeset
output.

Signed-off-by: Matthieu Herrb <matthieu@herrb.eu>
Reviewed-by: Mark Kettenis <kettenis@openbsd.org>
---
 man/xinit.man | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/xinit.man b/man/xinit.man
index f8005ce..de6e27c 100644
--- a/man/xinit.man
+++ b/man/xinit.man
@@ -117,7 +117,7 @@ This will start up a server named \fIX\fP, and will append the given
 arguments to the default \fIxterm\fP command.  It will ignore \fI\.xinitrc\fP.
 .TP 8
 .B "xinit \-e widgets \-\^\- ./Xorg \-l \-c"
-This will use the command \fI\.\/Xorg \-l \-c\fP to start the server and will
+This will use the command \fI\./Xorg \-l \-c\fP to start the server and will
 append the arguments \fI\-e widgets\fP to the default \fIxterm\fP command.
 .TP 8
 .B "xinit /usr/ucb/rsh fasthost cpupig \-display ws:1 \-\^\-  :1 \-a 2 \-t 5"
-- 
1.9.0

